flac.md

```diff
@@ -237,7 +237,7 @@ NOTES
   - 14 - During recording
   - 15 - During performance
   - 16 - Movie/video screen capture
-  - 17 - A bright coloured fish
+  - 17 - A bright colored fish
```

Comment:
It’s fine to be pedantic! All the other classics I noticed (e.g. -ise vs. -ize) are American English here, therefore I think it would be reasonable to unify to it.

(Charles Poynton always uses colour, coloured ;-)